codespell_lib/data/dictionary.txt

```diff
@@ -7776,6 +7776,8 @@ cylnder->cylinder
 cylnders->cylinders
 cylynders->cylinders
 cymk->CMYK
+cyphertext->ciphertext
+cyphertexts->ciphertexts
```

Comment:
I was _going_ to suggest that you could add `cypher->cipher` too, but it looks like cypher is an accepted (archaic?) variant spelling https://www.google.com/search?q=cypher+or+cipher (although not one I've ever seen myself) :wink: 
It seems `cyphertext` is definitely not a valid spelling though :+1: 

I wonder if it might by worth doing `cyphertext->ciphertext, hypertext,` ? (or perhaps that's "too different")

Reply:
> I was going to suggest that you could add cypher->cipher too, but it looks like cypher is an accepted (archaic?)

I preferred not to add `cypher` as misspell. Both `cipher` and `cypher` are the same word and same valid noun.
See https://www.oxfordlearnersdictionaries.com/definition/english/cypher?q=cypher

Although `cypher` is a valid word, but `cyphertext` is not a valid word in major online dictionaries (Cambridge, Collins, Merriam-Webster, Oxford Learners, dictionary.com). Except Wikipedia redirect `Cyphertext` to `Ciphertext`.

> I wonder if it might be worth doing 'cyphertext->ciphertext, hypertext,' ?

It would take 2 misspell mistakes to arrive with `hypertext`. IMO, almost all code developers who wrote `cyphertext` actually mean `ciphertext`. The percentage of those who mean `hypertext` would be negligible. I would add `hypertext` if I get more votes.